src/phases/summon-phase.ts

```diff
@@ -140,87 +291,90 @@ export class SummonPhase extends PartyMemberPokemonPhase {
-            globalScene.updateModifiers(this.isPlayer);
-            globalScene.updateFieldScale();
-
-            pokemon.showInfo();
-            pokemon.playAnim();
-            pokemon.setVisible(true);
-            pokemon.getSprite().setVisible(true);
-            pokemon.setScale(0.5);
-            pokemon.tint(getPokeballTintColor(pokemon.pokeball));
-            pokemon.untint(250, "Sine.easeIn");
-
-            globalScene.updateFieldScale();
-            tweens.add({
-              targets: pokemon,
-              duration: 250,
-              ease: "Sine.easeIn",
-              scale: pokemon.getSpriteScale(),
-              onComplete: () => {
-                pokemon.cry(pokemon.getHpRatio() > 0.25 ? undefined : { rate: 0.85 });
-                pokemon.getSprite().clearTint();
-                pokemon.resetSummonData();
-                // required to load the proper assets when loading from save data
-                if (pokemon.summonData.speciesForm) {
-                  pokemon.loadAssets(false);
-                }
-                time.delayedCall(1000, () => this.end());
-              },
-            });
-          },
-        });
-      },
+    const pokeBallYAnimation = async () => {
+      await playTween({
+        targets: pokeball,
+        duration: 150,
+        ease: "Cubic.easeOut",
+        y: (this.isPlayer ? 70 : 34) + fpOffset[1],
+      });
+
+      await playTween({
+        targets: pokeball,
+        duration: 500,
+        ease: "Cubic.easeIn",
+        y: (this.isPlayer ? 132 : 86) + fpOffset[1],
+      });
+    };
+
+    await Promise.allSettled([pokeBallXAnimation, pokeBallYAnimation()]);
+
+    globalScene.audioManager.playSound("se/pb_rel");
+    pokeball.destroy();
+    add.existing(pokemon);
+    field.add(pokemon);
+
+    if (!this.isPlayer) {
+      const playerPokemon = globalScene.getPlayerPokemon() as Pokemon;
+      if (playerPokemon?.isOnField()) {
+        field.moveBelow(pokemon, playerPokemon);
+      }
+      currentBattle.seenEnemyPartyMemberIds.add(pokemon.id);
+    }
+
+    animations.addPokeballOpenParticles(pokemon.x, pokemon.y - 16, pokemon.pokeball);
+    globalScene.updateModifiers(this.isPlayer);
+    globalScene.updateFieldScale();
+
+    pokemon.showInfo();
+    pokemon.playAnim();
+    pokemon.setVisible(true);
+    pokemon.getSprite().setVisible(true);
+    pokemon.setScale(0.5);
+    pokemon.tint(getPokeballTintColor(pokemon.pokeball));
+    pokemon.untint(250, "Sine.easeIn");
+
+    globalScene.updateFieldScale();
+
+    await playTween({
+      targets: pokemon,
+      duration: 250,
+      ease: "Sine.easeIn",
+      scale: pokemon.getSpriteScale(),
     });
+
+    pokemon.cry(pokemon.getHpRatio() > 0.25 ? undefined : { rate: 0.85 });
+    pokemon.getSprite().clearTint();
+    // required to load the proper assets when loading from save data
+    if (pokemon.summonData.speciesForm) {
```

Comment:
```suggestion
    // required to load the proper assets when loading from save data
    if (this.loaded && pokemon.summonData.speciesForm) {
```
this comment seems to imply that it's only needed when loading a save?